docs/release_notes.rst

```diff
@@ -126,6 +127,77 @@ Qiskit Metapackage Version  qiskit-terra  qiskit-aer  qiskit-ignis  qiskit-ibmq-
    For the ``0.7.0``, ``0.7.1``, and ``0.7.2`` meta-package releases the
    meta-package versioning strategy was not formalized yet.
 
+#############
+Qiskit 0.44.3
+#############
+
+.. _Release Notes_0.25.3:
+
+Terra 0.25.3
+============
+
+.. _Release Notes_Terra_0.25.3_Prelude:
+
+Prelude
+-------
+
+.. releasenotes/notes/prepare-0.25.3-155bf49f7a7ca309.yaml @ b'73433483ad4c5cb1010537eb52d4b406ef7a0446'
+
+Qiskit Terra 0.25.3 is a small patch release, fixing several bugs found in the 0.25 series.
+
+.. _Release Notes_0.25.2.1-19_Bug Fixes:
+
+Bug Fixes
+---------
+
+.. releasenotes/notes/10787-078d7caa70fc7de8.yaml @ b'1630b6e2987e5421e8d98119e7e137b93a5a6ca2'
+
+- Fix the Quantum Shannon Decomposition implemented in :func:`.qs_decomposition`. When a unitary
+  could not take advantage of the diagonal commutation optimization, it used to error.
+  Now, it leaves it as undecomposed 2-qubit unitary gate.
+  Fixes `#10787 <https://github.com/Qiskit/qiskit/issues/10787>`__
+
+.. releasenotes/notes/backend-estimator-v2-support-a698353aeeb5236c.yaml @ b'af48b85af9f65236f4c4568a92c34a896b1157da'
+
+- Fixed a bug where :class:`.primitives.BackendEstimator` cannot be used with the
+  :class:`.providers.BackendV2` that does not have coupling_map.
```

Comment:
```suggestion
  :class:`.providers.BackendV2` that does not have ``coupling_map``.
```

Reply:
I'll actually do this as

```suggestion
  :class:`.providers.BackendV2` that does not have any connectivity constraints.
```

I guess we could say :attr:`.BackendV2.coupling_map` is ``None`` too. But I think lets keep it higher level.